app/controllers/concerns/course/assessment/question/koditsu_question_code_insert_concern.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
```

Comment:
Should put this additional koditsu related file into services instead of our controllers.

See https://github.com/Coursemology/coursemology2/blob/6649e92c68cd719d140d66485e7ef9dbb26856ff/app/services/course/assessment/question/programming_codaveri/python/python_package_service.rb#L199

Reply:
done